zoom_monitor.py

```diff
@@ -293,6 +286,29 @@ def _spawn_browser_driver(chrome_options: Options):
         # process group instead.
         subprocess.Popen = functools.partial(
             subprocess_Popen, preexec_fn=lambda: os.setpgid(0, 0))
-    driver = Chrome(options=chrome_options)
+    driver = Chrome(options=_get_chrome_options())
     subprocess.Popen = subprocess_Popen  # Undo the monkey patch
     return driver
+
+def _get_chrome_options():
```

Comment:
We've got 50ish lines of browser-specific stuff that is unrelated to Zoom down here. Move it into a separate file.